data_diff/config.py

```diff
@@ -36,15 +37,17 @@ def _apply_config(config: Dict[str, Any], run_name: str, kw: Dict[str, Any]):
     else:
         run_name = "default"
 
-    if kw.get("database1") is not None:
+    # if db details are provided from cli that has precedence
+    if cli_options.database1 is not None:
         for attr in ("table1", "database2", "table2"):
-            if kw[attr] is None:
+            if cli_options.__getattribute__(attr) is None:
                 raise ValueError(f"Specified database1 but not {attr}. Must specify all 4 arguments, or neither.")
```

Comment:
```suggestion
                raise ValueError(f"Specified database1 but not {attr}. Must specify all 4 arguments (database1, table1, database2, table2), or none.")
```

Make it clear what 4 arguments are needed